client/components/themes-list/style.scss

```diff
@@ -9,13 +9,14 @@
 		max-width: 1000px;
 		margin: auto;
 	}
-}
 
-.themes-list__spacer {
-	width: 230px;
-	height: 0;
-	margin: 0 10px;
-	flex-grow: 1;
+	.pattern-assembler-cta-wrapper {
+		grid-column: 1/-1;
+		grid-row-start: 4;
```

Comment:
Works well! One minor detail, the PR description says:

> The styling of the CTA is also modified to match the style of theme showcase, CTA will be placed on 3rd row of the theme list

We should update the PR description if that's no longer the case, and placing the CTA in the 4th row instead 🙂